VisualStudio2015Community/Tools/AdminDeployment.xml

```diff
@@ -27,24 +37,21 @@
     <SelectableItemCustomization Id="SDKTools4Hidden" Selected="yes" FriendlyName="SDKTools4Hidden" />
     <SelectableItemCustomization Id="WCFDataServicesHidden" Selected="yes" FriendlyName="WCFDataServicesHidden" />
+    <SelectableItemCustomization Id="VSUV1" Hidden="no" Selected="no" FriendlyName="Visual Studio 2015 Update 1" />
     <SelectableItemCustomization Id="VSUV1PreReqV1" Selected="no" FriendlyName="Visual Studio 2015 Update 1 Prerequisite" />
     <SelectableItemCustomization Id="VSUV2PreReqV1" Selected="no" FriendlyName="Visual Studio 2015 Update 2 Prerequisite" />
-    <SelectableItemCustomization Id="NativeLanguageSupport_VCV1" Hidden="no" Selected="no" FriendlyName="Common Tools for Visual C++ 2015" />
-    <SelectableItemCustomization Id="NativeLanguageSupport_MFCV1" Hidden="no" Selected="no" FriendlyName="Microsoft Foundation Classes for C++" />
-    <SelectableItemCustomization Id="NativeLanguageSupport_XPV1" Hidden="no" Selected="no" FriendlyName="Windows XP Support for C++" />
-    <SelectableItemCustomization Id="Win81SDK_HiddenV1" Hidden="no" Selected="no" FriendlyName="Windows 8.1 SDK and Universal CRT SDK" />
+    <SelectableItemCustomization Id="MicroUpdateV2" Selected="no" FriendlyName="MicroUpdate 2.0 for Visual Studio 2015 Update 2" />
     <SelectableItemCustomization Id="FSharpV1" Hidden="no" Selected="no" FriendlyName="Visual F#" />
     <SelectableItemCustomization Id="PythonToolsForVisualStudioV5" Hidden="no" Selected="no" FriendlyName="Python Tools for Visual Studio (March 2016)" />
     <SelectableItemCustomization Id="ClickOnceV1" Hidden="no" Selected="no" FriendlyName="ClickOnce Publishing Tools" />
     <SelectableItemCustomization Id="SQLV1" Hidden="no" Selected="no" FriendlyName="Microsoft SQL Server Data Tools" />
-    <SelectableItemCustomization Id="PowerShellToolsV1" Hidden="no" Selected="no" FriendlyName="PowerShell Tools for Visual Studio" />
     <SelectableItemCustomization Id="SilverLight_Developer_KitV1" Hidden="no" Selected="no" FriendlyName="Silverlight Development Kit" />
-    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV9" Hidden="no" Selected="no" FriendlyName="Tools (1.3.1) and Windows 10 SDK (10.0.10586)" />
+    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV10" Hidden="no" Selected="no" FriendlyName="Tools (1.3.2) and Windows 10 SDK (10.0.10586)" />
     <SelectableItemCustomization Id="Win10_EmulatorV1" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10240)" />
     <SelectableItemCustomization Id="Win10_EmulatorV2" Hidden="no" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10586)" />
-    <SelectableItemCustomization Id="XamarinVSCoreV2" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.3)" />
+    <SelectableItemCustomization Id="XamarinVSCoreV3" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.4)" />
```

Comment:
This is the newer admin file - notice `XamarinVSCoreV3` 4.0.4 vs 4.0.3 and above `Windows10_ToolsAndSDKV10` 1.3.2